cmd/kubectl/kubectlcobra/inventory.go

```diff
@@ -81,3 +82,99 @@ func (i *Inventory) String() string {
 		i.GroupKind.Group, fieldSeparator,
 		i.GroupKind.Kind)
 }
+
+// InventorySet encapsulates a grouping of unique Inventory
+// structs. Organizes the Inventory structs with a map,
+// which ensures there are no duplicates. Allows set
+// operations such as merging sets and subtracting sets.
+type InventorySet struct {
```

Comment:
Maybe it doesn't make a big difference, but could this be `type InventorySet map[string]*Inventory` instead of wrapping the map?